apps/web/pages/api/recorded-daily-video.ts

```diff
@@ -105,31 +141,36 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
       },
     });
 
+    const response = await getDownloadLinkOfCalVideoByRecordingId(recordingId);
+    const downloadLinkResponse = downloadLinkSchema.parse(response);
+    const downloadLink = downloadLinkResponse.download_link;
+
+    const evt: CalendarEvent = {
+      type: booking.title,
```

Comment:
This should be booking.eventType.title I think. But it's wrongly set at many places.
<img width="965" alt="Screenshot 2023-05-09 at 10 41 07 PM" src="https://github.com/calcom/cal.com/assets/1780212/0a73f82c-0103-4938-a633-939c18087ca4">